composer/algorithms/factorize/factorize_core.py

```diff
@@ -18,23 +18,26 @@ class LowRankSolution:
     to the original output space.
 
     Args:
-        Wa: First linear operation in the factorized approximation. For a
+        Wa (:class:`torch.Tensor`, optional): First linear operation in the 
+            factorized approximation. For a
             factorized linear operation, ``Wa`` is a matrix. For a factorized
             convolution, ``Wa`` matches the shape of the convolution's
             original weight parameter, except along the channel axis.
-        Wb: Second linear operation in the factorized approximation. Shape
+        Wb (:class:`torch.Tensor`, optional): Second linear operation in the
+            factorized approximation. Shape
             is such that composing ``Wb`` with ``Wb`` yields an output of
             the same size as the original operation.
-        bias: vector added to the output of the second linear operation
-        rank: output dimensionality (channels or features) of the first linear
-            operation, and input dimensionality of the second input operation.
-        nmse: normalized mean squared error obtained during the optimization
+        bias (:class:`torch.Tensor`, optional): vector added to the output of 
+            the second linear operation.
+        rank (int, optional): output dimensionality (channels or features) of the first linear
+            operation, and input dimensionality of the second input operation. Default: ``-1``.
+        nmse (float, optional): normalized mean squared error obtained during the optimization
```

Comment:
really nitpicky, but maybe shorter lines here for consistency with other docstrings?

Reply:
okay, fixed!